qemu/tests/vsock_seqpacket_test.py

```diff
@@ -0,0 +1,114 @@
+import os
+import random
+import time
+
+import aexpect
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_net
+
+
+def copy_compile_testsuite(test, vm, session):
+    """
+    Download and compile upstream test suite on guest and host.
+
+    :param test: QEMU test object
+    ::param vm: Object qemu_vm.VM
+    :param session: Guest session object
+    """
+    vsock_test_base_dir = "/home/"
```

Comment:
Maybe can save it into the `test.workdir` or `test.tmpdir`?

Reply:
Considering use same directory for both host and guest for the following operations, keep use /home here.